pkg/api/deploymentapi/deptemplateapi/list_test.go

```diff
@@ -152,6 +155,7 @@ func TestList(t *testing.T) {
 							"region":                       []string{"us-east-1"},
 							"show_instance_configurations": []string{"true"},
 							"show_hidden":                  []string{"false"},
+							"show_max_zones":               []string{"false"},
```

Comment:
Looks like we test only cases when `show_max_zones` is `false` (for both get and list). Does it make sense to cover the `true` case as well?

Reply:
Added true case tests.